tests/inte_test/test_mvlstm.py

```diff
@@ -0,0 +1,91 @@
+import os
+import shutil
+
+import numpy as np
+import pytest
+
+import matchzoo as mz
+
+
+@pytest.fixture(scope='module')
+def train_data():
```

Comment:
It's nice to have unit tests but it's not required. If any of these tests are failing you may simply delete them and I will add tests for them (together with other models) later.

Reply:
OK, I will fix them.